src/ArticleHolder.jsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from "react";
+import Articalizer from "./Articalizer";
+
+export default function ArticleHolder() {
+  const [articles, setArticles] = useState([]);
+  const [mostCommentedCooking, setMostCommentedCooking] = "";
+  const [mostCommentedCoding, setMostCommentedCoding] = "";
+  const [mostCommentedFootball, setMostCommentedFootball] = "";
+
+  useEffect(() => {
+    fetch("https://nc-news-itve.onrender.com/api/articles")
```

Comment:
Could we refactor this to axios and extract it to it's own api.js file please? Will make life much easier as we get further into the project!